core/src/main/java/org/apache/calcite/rel/logical/LogicalTableModify.java

```diff
@@ -92,4 +93,8 @@ public static LogicalTableModify create(RelOptTable table,
         sole(inputs), getOperation(), getUpdateColumnList(),
         getSourceExpressionList(), isFlattened());
   }
+
+  @Override public RelNode accept(RelShuttle shuttle) {
+    return shuttle.visit(this);
+  }
```

Comment:
Could you please check whether other operators have the same issue too?

Reply:
Other operators don't have the same issue. These two operators had overloaded methods in `RelShuttle` but didn't have overridden `accept()` methods. Other logical operators neither had overloaded methods in `RelShuttle` nor `accept()` methods (or have both options).

Reply:
Get it! Thank you for your effort.